src/config.js

```diff
@@ -86,12 +85,12 @@ export function newConfig() {
         this._bidderTimeout = val;
       },
 
-      // domain where prebid is running for cross domain iframe communication
-      _publisherDomain: DEFAULT_PUBLISHER_DOMAIN,
+      _publisherDomain: null,
       get publisherDomain() {
         return this._publisherDomain;
       },
       set publisherDomain(val) {
+        logWarn('publisherDomain is deprecated and has no effect since v7 - use pageUrl instead')
```

Comment:
what about https://github.com/prebid/Prebid.js/blob/e47b9081ef3f6283053c46dd432a6d593d9b3558/src/prebid.js#L951 and https://github.com/prebid/Prebid.js/blob/130f7265cf4baa470d62102993217c3dd69998b2/test/spec/config_spec.js#L154

Reply:
Updated the comment; the test should stay IMO as get/set of publisherDomain is still a special case that's not covered by other tests.
It was my intention to remove the default value for it, yes, as it'd be odd to have it default to something when the goal is to remove its use.